src/main/python/resources/imputers/capice_imputing.py

```diff
@@ -151,17 +154,9 @@ def impute(self, datafile: pd.DataFrame):
         datafile.fillna(self.impute_values, inplace=True)
         datafile = datafile.astype(dtype=self.pre_dtypes, copy=False)
         datafile = datafile.astype(dtype=self.dtypes, copy=False)
-        datafile = self._add_missing_columns(datafile)
         self.log.info('Imputing successfully performed.')
         return datafile
 
-    @deprecated
-    def _add_missing_columns(self, datafile: pd.DataFrame):
-        for key, value in self.impute_values.items():
-            if key not in datafile.columns:
-                datafile[key] = value
-        return datafile
-
     def _correct_dtypes(self, datafile: pd.DataFrame):
```

Comment:
As discussed in regards to #46:
- Add `CRITICAL` logger message when a required training column is missing.
- Update README with more information about training a model (such as which fields are required).
- Consider validating whether all required fields are present during processing of input file (to reduce waiting time before the error occurs).